dev/source/docs/simulation-2.rst

```diff
@@ -6,11 +6,26 @@ Simulation
 
 .. image:: ../images/simulation-landingpage.jpg
 
-Simulation allows for safe testing of experimental code and settings and crashing virtual vehicles is a lot cheaper than crashing real ones!
+Simulation allows for safe testing of experimental code and settings, and crashing virtual vehicles is a lot cheaper than crashing real ones!
+
+Overview
+========
+
+Simulation is implemented by using a Flight Dynamics Model (FDM) of the vehicle to simulate the physics involved with vehicle movement, which receives inputs from a SITL (Software in the Loop) program running the ArduPilot firmware (which are the firmware's servo/motor outputs) and outputs vehicle status,position, velocities,etc. that result from those inputs back to the firmware sim just like sensors would in the real world.
```

Comment:
This is a bit of a run-on sentence.  Probably pay to break it up.

Reply:
broken up